src/priorityq.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+from __future__ import division
+
+
+class PriorityQueue(object):
+
+    def __init__(self):
+        """Create a priority queue."""
+        self._container = []
+
+    def insert(self, item):
+        """Insert item into priority queue."""
+        self._container.append(item)
+        self._sort_up(len(self._container))
+
+    def pop(self):
+        """Remove and return highest priority item in priority queue."""
+        if len(self._container) == 1:
+            return self._container.pop()
+        elif len(self._container) > 1:
+            return_value = self._container[0]
+            self._container[0] = self._container.pop()
+            self._sort_down(0)
+            return return_value
+        else:
+            raise IndexError("Priority queue empty")
+
+    def peek(self):
+        """Return highest priority item in priority queue without removing it."""
+        return self._container[0]
+        pass
+
+    def _sort_down(self, index):
+        # helper method to maintain heap property after popping
```

Comment:
Be consistent with your docstrings. The triple quote style used on the other functions is preferred, even if these are just helper methods. Reserve `#` comments for block or line level comments.